setup.py

```diff
@@ -52,4 +52,9 @@ def read(fname):
       },
       install_requires=[
         "PrettyTable>=0.7.2",
-        "fasteners"])
+          "fasteners"
+      ],
+      extras_require = {
```

Comment:
I'm not all that familiar with `extras_require`. Will it always try to install the packages and just not blow up if it fails?

Reply:
It allows the `"mbed-ls[colorized_log]"`, but the code will Just Work (tm) if it can import the `colorlog` package.

Reply:
Cool 👍